### ISSUE TITLE
Error in plotting

### ISSUE THREAD

**Issue by Thexiyang** [gatech-genemark/Mgcod#2]
During running this software, I came across this issue. Any suggestion on how to solve this? Thanks！

```
Traceback (most recent call last):
  File "/Mgcod/bin/mgcod.py", line 185, in <module>
    main(sys.argv[1:])
  File "Mgcod/bin/mgcod.py", line 152, in main
    if pipeline.predicted_switch_regions[contig] is None:
TypeError: 'NoneType' object is not subscriptable

```

**Comment by arpfennig** [comment]
Hi, 

Could be a problem with the pandas or numpy version. Which versions are you using?

If you could share your input file and the command you used, I can try to reproduce the error and see what's causing it.

Best,
Aaron

**Comment by arpfennig** [comment]
Closing due to inactivity. Please, re-open if this issue persists.

**Comment by shujieH** [comment]
Hi,  Aaron

Thanks for providing us with this excellent tool! During the runtime, I met the same problem as above. And the versions of 

pandas numpy, which I used were v1.2.4 and v1.18.1, respectively. Here is my input file and input command. Please help me 

solve this problem. Thank you!

[test.zip](https://github.com/gatech-genemark/Mgcod/files/12455525/test.zip)

mgcod.py -i test.fa -r --isoforms -v -m ./plots -p test_mgm_result -o test_results

![捕获](https://github.com/gatech-genemark/Mgcod/assets/56304053/58637384-cd50-4dd9-9b71-0e87246dd4bc)



Best wishes,
Shujie.

**Comment by arpfennig** [comment]
Hi Shujie,

Thanks for using Mgcod and glad to hear that you find useful.

What version of Mgcod are you running? I could reproduce your error with version 1.0.0. However, it seems to be fixed in version 1.0.1. Depending on how you installed Mgcod, you can update it by running `conda update mgcod` or `git pull`.

Let me know if the issue persists after updating Mgcod.

Best,
Aaron

**Comment by shujieH** [comment]
Hi, Aaron

Thank you for your prompt reply!

According to your suggestion, I installed the latest version of Mgcod (v1.0.1) with conda, and finally solved the above problem smoothly. Thanks again for your kind help.

All Best, 
Shujie
